### DISCUSSION TITLE
want to parse the json output from `turbo build --filter=[HEAD^] --dry=json

### DISCUSSION THREAD

**Discussion by shkdev94** [vercel/turborepo#7694] [Help] [answered]
### Summary

i have used jq to parse json and i want to parse the json output from  `turbo build --filter=[HEAD^] --dry=json`. but i cant. because when i run `turbo build --filter=[HEAD^] --dry=json`, output include additional strings which is not json format like this.

```bash

// additional strings
> project-name@ build /Users/username/projects
> turbo build --filter=[HEAD^] --dry=json

{
 ...json format
}
```

is there any other solutions ??

thank you

**Comment by mehulkar** [comment, marked as answer]
That output looks like it's from a package manager like `pnpm` or `npm`. Are you sure you're running `turbo build --dry=json` or are you running this through your package manager?

**Reply by shkdev94** [reply]
oh right. when use `turbo build --dry=json`, there is no additional strings. thank you

it's just my opinion, but it would be nice to print the same output when using package manager

**Reply by mehulkar** [reply]
It would be, but the behavior of wrapping tools is not within turbo's control. It would be the same issue if you wrote a shell script that calls turbo and has some `echo` statements before/after!
